interface.js

```diff
@@ -0,0 +1,17 @@
+// interface.js
+document.addEventListener("DOMContentLoaded", () => {
+  fetch('https://content.guardianapis.com/politics?api-key=41355124-6ce9-43d0-99af-4b99580d961a')
```

Comment:
You can use an environment variable to hide api key

Reply:
To make this easier to read, you can refactor the steps into its own functions such as `getArticleData()`,  `renderArticleLink(article data)`. 
example: https://github.com/makersacademy/course/blob/main/pills/calling_apis_in_javascript.md

Reply:
I agree it does need to be refactored so it makes more sense